pkg/kubernetes/scripts/images.js

```diff
@@ -356,6 +356,9 @@
              */
             select.register("taggedBy", function(tag) {
                 var i, len, results = { };
+                // catch condition when tag.items is null due to imagestream import error
+                if (!tag.items)
+                    tag.items = [];
```

Comment:
I think this should just ```return select(null);```. We shouldn't be modifying the tag after it's been loaded.